### DISCUSSION TITLE
是否有人也发现，新版调用大模型如 gpt或 claude 模型回答效果变差了

### DISCUSSION THREAD

**Discussion by wangxiaodong1021** [lobehub/lobehub#4743] [General Question] [answered]
最近半个多月使用最新的 lobe-chat,发现回答效果明显变差了，相同的模型，用其他的工具平台调用依然正常，但是用这个平台感觉就很糟糕的答案。不只是我自己这样的直观感受，几个人都反应了类似情况，是否是默认的一些调用参数被变动了？目前不得不临时迁移到其他平台。

**Comment by arvinxx** [comment]
能否给出具体的例子？参数近期并没有改过。

如果有具体的例子我可以看看。

目前我自用的情况下并没有感觉变差，还是需要详细的例子分析看看的。如果觉得没法公开发，可以在 discord 上私戳我

**Comment by SoulGolf** [comment]
+1,有可能是代理商浑水摸鱼或者被降智了,代码方面还不如deepseek好用.

**Comment by wangxiaodong1021** [comment]
应该不是，我在两个供应商的api调用都有类似的结果，但对应的api放别的平台app用就感觉又正常了

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: soulnull ***@***.***>
发送时间: Wednesday, November 20, 2024 9:10:15 PM
收件人: lobehub/lobe-chat ***@***.***>
抄送: Wang Xiaodong ***@***.***>; Author ***@***.***>
主题: Re: [lobehub/lobe-chat] 是否有人也发现，新版调用大模型如 gpt或 claude 模型回答效果变差了 (Discussion #4743)


+1,有可能是代理商浑水摸鱼或者被降智了,代码方面还不如deepseek好用.

―

**Comment by arvinxx** [comment, marked as answer]
和 @wangxiaodong1021 私聊确认了下，应该是前段时间 #4639 上线默认开启了限制聊天上下文导致（老用户可能之前的配置是 1 ），把值调大，或者关掉上下文限制就好了（如下图）。

![image](https://github.com/user-attachments/assets/60dc7b78-7536-4ce2-9f5b-8cf09db87b99)

**Reply by iparanoid** [reply]
请问有没有变量可以设置，默认关闭「限制历史消息数」

**Reply by arvinxx** [reply]
可以设置 DEFAULT_AGENT_CONFIG 详见文档
